### PR TITLE
Remove link to Ember CLI testing pages, added content on testem

### PR DESCRIPTION
**Pull request by joostdevries**
#410

### CODE REVIEW

File: source/testing/index.md [hunk 1]
```diff
@@ -19,27 +19,28 @@ the example scenario above, some acceptance tests you might write are:
...
-[QUnit] is the default testing framework for this guide, but others are 
+[QUnit] is the default testing framework for this guide, but others are
 supported through third-party addons.
 
 ### How to Run Your Tests
 
-Run your tests with `ember test` on the command-line. You can re-run your tests on every file-change with `ember test --server`. For more details and options, see [Ember CLI – Testing](http://www.ember-cli.com/user-guide/#testing) and `ember help test`.
+Run your tests with `ember test` on the command-line. You can re-run your tests on every file-change with `ember test --server`. These commands run your tests using [Testem] to make testing multiple browsers very easy. You can configure Testem using the `testem.json` file in your application root.
+
+Alternatively you can run the tests in your regular browser using the QUnit interface. Run `ember server` and navigate to `http://localhost:4200/tests`. When the app runs in /tests it runs in the development environment, not the test environment.
```
Comment (michaelrkn) [inline source/testing/index.md]: We've been talking about trying to simplify the guides to just talk about the recommended happiest path. In that spirit, I'd suggest we take out this paragraph.

Comment (locks) [inline source/testing/index.md]: +1

Comment (michaelrkn) [inline source/testing/index.md]: ping @joostdevries

**Comment by joostdevries** [comment]
updated.

**Comment by michaelrkn** [comment]
Looks great! Thanks!

**Pull request merged**